packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -1,18 +1,199 @@
+   * @param {string} phoneNumber Phone Number
+   * @param {string} account Account
+   */
+  getAttestationStat = proxyCall(
+    this.contract.methods.getAttestationStats,
+    tupleParser(PhoneNumberUtils.getPhoneHash, (x: string) => x),
+    (stat) => ({ completed: toNumber(stat[0]), total: toNumber(stat[1]) })
+  )
+
+  /**
+   * Returns the set wallet address for the account
+   * @param {string} account Account
+   */
+  getWalletAddress = proxyCall(this.contract.methods.getWalletAddress)
+
+  /**
+   * Sets the data encryption of the account
+   * @param {string} encryptionKey The key to set
+   */
   setAccountDataEncryptionKey = proxySend(
     this.kit,
     this.contract.methods.setAccountDataEncryptionKey
   )
 
+  /**
+   * Sets the wallet address for the account
+   * @param {string} address The address to set
+   */
+  setWalletAddress = proxySend(this.kit, this.contract.methods.setWalletAddress)
+
+  /**
+   * Calculates the amount of CeloToken to request Attestations
+   * @param {CeloToken} token The token to pay for attestations for
+   * @param {number} attestationsRequested The number of attestations to request
+   */
+  async approveAttestationFee(token: CeloToken, attestationsRequested: number) {
+    const tokenContract = await this.kit.contracts.getContract(token)
+    const fee = await this.attestationFeeRequired(token, attestationsRequested)
+    return tokenContract.approve(this.address, fee.toString())
+  }
+
+  /**
+   * Approves the transfer of CeloToken to request Attestations
+   * @param {CeloToken} token  The token to pay for attestations for
+   * @param {number} attestationsRequested  The number of attestations to request
+   */
+  async attestationFeeRequired(token: CeloToken, attestationsRequested: number) {
+    const tokenAddress = await this.kit.registry.addressFor(token)
+    const attestationFee = await this.contract.methods.getAttestationRequestFee(tokenAddress).call()
+    return new BigNumber(attestationFee).times(attestationsRequested)
+  }
+
+  /**
+   * Returns an array of attestations that can be completed, along with the issuers public key
+   * @param {string} phoneNumber
+   * @param {string} account
+   */
+  async getActionableAttestations(
+    phoneNumber: string,
+    account: string
+  ): Promise<ActionableAttestation[]> {
+    const phoneHash = PhoneNumberUtils.getPhoneHash(phoneNumber)
+    const expirySeconds = await this.attestationExpirySeconds()
+    const nowInUnixSeconds = Math.floor(new Date().getTime() / 1000)
+
+    const issuers = await this.contract.methods.getAttestationIssuers(phoneHash, account).call()
+    const issuerState = Promise.all(
+      issuers.map((issuer) =>
+        this.contract.methods
+          .getAttestationState(phoneHash, account, issuer)
+          .call()
+          .then(parseAttestationInfo)
+      )
+    )
+
+    // Typechain is not properly typing getDataEncryptionKey
+    // @ts-ignore
```

Comment:
my recommendation here is to do `(somethingWithIncorrectType as any)` so the rest of the line is validated.